src/zelos/config_gen.py

```diff
@@ -138,8 +141,65 @@ def generate_parser():
         "--traceon",
         type=str,
         default="",
-        help="[Experimental] Enable verbose tracing after specified address "
```

Comment:
can `traceon` and `traceoff` be removed?

Reply:
Not yet. Feeds are not being used anywhere at the moment. Once we replace verbose mode we can use these flags instead of traceon and traceoff